ocw/lib/cleanup.py

```diff
@@ -32,5 +32,23 @@ def cleanup_run():
                 send_mail('{} on Cleanup in [{}]'.format(type(e).__name__, vault_namespace), traceback.format_exc())
 
 
+def list_clusters():
+    cfg = ConfigFile()
+    if cfg.has('clusters'):
+        for vault_namespace in cfg.getList(['cleanup', 'namespaces'], ['']):
+            try:
+                orphaned_clusters = EC2(vault_namespace).list_clusters()
+                logger.info("%d clusters found", len(orphaned_clusters))
+                if len(orphaned_clusters):
+                    clusters_str = ' '.join([str(cluster) for cluster in orphaned_clusters])
+                    logger.debug("Full clusters list - %s", clusters_str)
+                    send_mail("EC2 orphaned clusters found", clusters_str)
```

Comment:
I think this mail should only go to the corresponding namespace.

Reply:
https://github.com/cfconrad/pcw/blob/8ae75a70b43b00d806500b0bf2b3cc2900603052/ocw/lib/emailnotify.py#L54